singer/transform.py

```diff
@@ -55,19 +56,23 @@ class SchemaKey:
     any_of = 'anyOf'
 
 class Error:
-    def __init__(self, path, data, schema=None):
+    def __init__(self, path, data, schema=None, logging_level=logging.INFO):
         self.path = path
         self.data = data
         self.schema = schema
+        self.logging_level = logging_level
 
     def tostr(self):
         path = ".".join(map(str, self.path))
         if self.schema:
             msg = "does not match {}".format(self.schema)
```

Comment:
Can you move the data key from below to here? That way the logging comes out like:

```
thing.foo: data does not match ...schema
thing.foo: not in schema ...schema
```